src/test/resources/epub3/navigation-publication.feature

```diff
@@ -71,14 +71,12 @@ Feature: EPUB 3 ▸ Navigation Document ▸ Full Publication Checks
     When checking EPUB 'nav-page-list-reading-order-valid'
     Then no errors or warnings are reported
 
-  Scenario: Report a `page-list nav` whose links do not match the spine order 
+  Scenario: Verify a `page-list nav` whose links do not match the spine order 
     When checking EPUB 'nav-page-list-unordered-spine-warning'
-    Then warning NAV-011 is reported
     And no other errors or warnings are reported
```

Comment:
I guess this doesn't matter much practically for processing, but shouldn't this line start with "Then no errors..." instead of "And no other errors..."?

(Same would apply to line 80.)

Reply:
Yeah, all the [steps keywords](https://cucumber.io/docs/gherkin/reference/#keywords) can be used interchangeably, so it’s mostly a matter of taste. I personally find “Then warning X is reported AND no other errors or warnings are reported” reads more naturally than repeating “Then”, but YMMV 😊

Reply:
oh, no I understand just now. What I have is "When … And …" when it would obviously read better as "When … Then …", you're absolutely right.
Filed as #1386